client/tests/test_nodes.py

```
@@ -0,0 +1,132 @@
+            eth_rpc_url="http://localhost:8545",
+            aggregator_server_ip_port_address="localhost:8090",
+            ecdsa_private_key_store_path="tests/keys/operator.ecdsa.key.json",
+            auto_update=False,
+            nodes=nodes_config,
+        )
+    )
+    # sync nodes with the updated config
+    operator.nodes_manager.sync_nodes()
+
+    # get the nodes list from the server and verify the updates
+    expected_result = get_expected_nodes(nodes_config)
```

Flake8 (7.3.0)

[error] local variable 'expected_result' is assigned to but never used

(F841)